R/tidy-upkeep.R

```diff
@@ -30,16 +68,67 @@ use_tidy_upkeep_issue <- function(year = NULL) {
   view_url(issue$html_url)
 }
 
-upkeep_checklist <- function(year = NULL,
-                             posit_pkg = is_posit_pkg(),
-                             posit_person_ok = is_posit_person_canonical()) {
-  year <- year %||% 2000
+upkeep_checklist <- function(year = NULL) {
+
+  bullets <- c(
+    year,
+    "",
+    todo("`usethis::use_readme_rmd()`", !file_exists("README.Rmd")),
+    todo("`usethis::use_roxygen_md()`", !is_true(uses_roxygen_md())),
+    todo("`usethis::use_github_links()`", !has_github_links()),
+    todo("`usethis::use_pkgdown_github_pages()`", !uses_pkgdown()),
+    todo("`urlchecker::url_check()`"),
+    todo(
+      "
```

Comment:
Nitpick: I think the `todo()` below ("Align the names ...") uses styling more consistent with what's elsewhere in usethis.